### PR TITLE
[gui] use CVariant on SetVisualisation(...)

### PR DESCRIPTION
**Pull request by AlwinEsch**
This is a part what was needed on my add-on system rework to prevent compile error, is independent and make sense for me to have.

### CODE REVIEW

File: xbmc/music/dialogs/GUIDialogVisualisationPresetList.cpp [hunk 1]
```diff
@@ -61,7 +62,8 @@ void CGUIDialogVisualisationPresetList::SetVisualisation(CVisualisation* vis)
   {
     SetUseDetails(false);
     SetMultiSelection(false);
-    SetHeading(StringUtils::Format(g_localizeStrings.Get(13407).c_str(), m_viz->Name().c_str()));
+    std::string heading = StringUtils::Format(g_localizeStrings.Get(13407).c_str(), m_viz->Name().c_str());
+    SetHeading(CVariant{heading});
```
Comment (Paxxi) [inline xbmc/music/dialogs/GUIDialogVisualisationPresetList.cpp]: Shouldn't be a need for this temporary

**Comment by Paxxi** [comment]
:thumbs_up: one less missing include is always a good thing

File: xbmc/music/dialogs/GUIDialogVisualisationPresetList.cpp (hunk 1 above)
Comment (AlwinEsch) [inline xbmc/music/dialogs/GUIDialogVisualisationPresetList.cpp]: Was a c&p from the other places where it is used, thought is wanted this way ;-)

**Comment by AlwinEsch** [comment]
jenkins build this please

File: xbmc/music/dialogs/GUIDialogVisualisationPresetList.cpp (hunk 1 above)
Comment (Paxxi) [inline xbmc/music/dialogs/GUIDialogVisualisationPresetList.cpp]: Both are fine really, when doing the original I mainly chose for readability so there's probably a mix.

**Comment by AlwinEsch** [comment]
Ok to merge in?

**Pull request merged**